frontend/src/components/entity/EntityCardTitle.tsx

```diff
@@ -27,6 +27,7 @@ const EntityCardTitle = ({
       color="text.primary"
       lineHeight="1.3"
       sx={{
+        overflowWrap: 'anywhere',
```

Comment:
This change prevents the two entity cards displayed on the comparison page to have different width on very small screens. It also fixes a problem we have in production:

**in main at** 12e2f2c57bcbcb6c02615d4303fcafa4ad8c0c16 (see the title of the second card)

![caire](https://github.com/tournesol-app/tournesol/assets/39056254/20d5bdfa-79a8-48b1-b234-e9b20b5dbefd)

**in this branch**

![capture2](https://github.com/tournesol-app/tournesol/assets/39056254/9fddcc61-4cca-4310-b7a8-162355b995cb)